rules.ini

```
@@ -161,7 +161,8 @@ AntiCheatExpert[] = (?:^|/)AceAntibotClient/
 Arbiter = (?:^|/)Arbiter\.dll$
 BattlEye = (?:^|/)BEService(?:_x64)?\.exe$
 BlackCipher = (?:^|/)BlackCipher/
-EasyAntiCheat = (?:^|/)EasyAntiCheat/
+EasyAntiCheat[] = (?:^|/)EasyAntiCheat/
+EasyAntiCheat[] = (?:^|/)EasyAntiCheat_Setup\.exe$
```

Let's not make things too hard for nothing, we can just look for EasyAntiCheat, we probably won't ever get any false positive
```suggestion
EasyAntiCheat[] = EasyAntiCheat
```